Test/overall_test.py

```diff
@@ -1393,4 +1393,29 @@
 2
 >>> cm.label_map[2]
 3
+>>> y_act = [0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2]
+>>> y_pre = [0,0,0,0,0,0,0,0,0,1,1,1,0,0,0,1,1,1,1,1,2,0,1,2,2,2,2]
+>>> cm = ConfusionMatrix(y_act,y_pre)
+>>> cm.to_array()
+array([[9, 3, 0],
+       [3, 5, 1],
+       [1, 1, 4]])
+>>> cm.to_array(normalized=True)
+array([[0.75   , 0.25   , 0.     ],
+       [0.33333, 0.55556, 0.11111],
+       [0.16667, 0.16667, 0.66667]])
+>>> cm.to_array(one_vs_all=True)
+array([[9, 3, 0],
+       [3, 5, 1],
+       [1, 1, 4]])
```

Comment:
I think here we should raise an warning about none given `class_name`.

Reply:
It's not a big problem, we will add a warning to `normalized_table_calc` function later (another issue/PR)